src/Compilers/Core/AnalyzerDriver/AnalyzerExecutor.cs

```diff
@@ -495,8 +495,9 @@ internal static bool IsCanceled(Exception ex, CancellationToken cancellationToke
         internal static Diagnostic GetAnalyzerExceptionDiagnostic(DiagnosticAnalyzer analyzer, Exception e)
         {
             var descriptor = new DiagnosticDescriptor(AnalyzerExceptionDiagnosticId,
-                AnalyzerDriverResources.AnalyzerFailure,
-                AnalyzerDriverResources.AnalyzerThrows,
+                title: AnalyzerDriverResources.AnalyzerFailure,
+                messageFormat: AnalyzerDriverResources.AnalyzerThrows,
+                description: string.Format(AnalyzerDriverResources.AnalyzerThrowsDescription, e.ToString()),
```

Comment:
@mavasani I actually have one suggestion. can you make sure it flattens all exceptions (in case it has inner exceptions) and show stack for all of them?